src/aria/widgets/form/TextInput.js

```diff
@@ -910,9 +912,13 @@ module.exports = Aria.classDefinition({
          * input widget if the autoselect property has been set to true.
          * @protected
          */
-        _dom_onclick : function () {
+        _dom_onclick : function (event) {
             if (!!this._cfg.onclick) {
-                this.evalCallback(this._cfg.onclick);
+                var evtWrapper = new ariaTemplatesDomEventWrapper(event);
+                this.evalCallback(this._cfg.onclick, evtWrapper);
+                if (evtWrapper) {
+                    evtWrapper.$dispose();
+                }
```

Comment:
I don't see how `evtWrapper` could be falsy here... so I would remove the condition here (and also in the next two cases) and call `evtWrapper.$dispose()` unconditionally.